packages/full-stack-tests/src/hierarchy-builder/LabelsFormatting.test.ts

```diff
@@ -0,0 +1,838 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { expect } from "chai";
+import { Subject } from "@itwin/core-backend";
+import { Guid, Id64 } from "@itwin/core-bentley";
+import { IModel, Rank } from "@itwin/core-common";
+import { IModelConnection } from "@itwin/core-frontend";
+import { SchemaContext } from "@itwin/ecschema-metadata";
+import { ECSchemaRpcLocater } from "@itwin/ecschema-rpcinterface-common";
+import { createECSqlQueryExecutor, createMetadataProvider, createValueFormatter } from "@itwin/presentation-core-interop";
+import {
+  createConcatenatedValueSelector,
+  createPropertyValueSelector,
+  HierarchyProvider,
+  IHierarchyLevelDefinitionsFactory,
+  IPrimitiveValueFormatter,
+  NodeSelectClauseFactory,
+} from "@itwin/presentation-hierarchy-builder";
+import { julianToDateTime } from "@itwin/presentation-hierarchy-builder/lib/cjs/hierarchy-builder/internal/Common";
+import {
+  buildIModel,
+  importSchema,
+  insertDrawingCategory,
+  insertDrawingGraphic,
+  insertDrawingModelWithPartition,
+  insertPhysicalElement,
+  insertPhysicalModelWithPartition,
+  insertPhysicalPartition,
+  insertPhysicalSubModel,
+  insertSpatialCategory,
+} from "../IModelUtils";
+import { initialize, terminate } from "../IntegrationTests";
+import { validateHierarchy } from "./HierarchyValidation";
+
+describe("Stateless hierarchy builder", () => {
+  describe("Labels formatting", () => {
```

Comment:
```suggestion
  describe("labels formatting", () => {
```

Reply:
We start names of those high level describes with upper case. I checked to confirm with other full stack tests and that pattern used across all of them.